include/ppx/application.h

```diff
@@ -357,7 +355,7 @@ struct ApplicationSettings
         bool                     listGpus              = false;
         std::string              metricsFilename       = "report_@.json";
         bool                     overwriteMetricsFile  = false;
-        std::pair<int, int>      resolution            = std::make_pair(0, 0);
+        std::pair<int, int>      resolution            = std::make_pair(1280, 720);
```

Comment:
I think this modifies things for XR. Currently I think we just pass 0 as the default, and XR chooses suggested resolution https://github.com/google/bigwheels/blob/a99d5bfbd4127f41677fe7113447aae587e90d4f/include/ppx/xr_component.h#L179 
which I think is a good default behaviour.
However, with this I think it would choose 1280x720 instead?